src/lib/loaders/api/index.ts

```diff
@@ -121,6 +126,7 @@ export default (opts) => ({
       requestIDs: opts.requestIDs,
       userAgent: opts.userAgent,
       appToken: opts.appToken,
+      isMutation: opts.isMutation,
```

Comment:
For now, passing this option through for this behavior just for gravity loaders.